src/compiler/diagnosticMessages.json

```diff
@@ -671,6 +671,10 @@
         "category": "Error",
         "code": 1215
     },
+    "Identifier expected. '__esModule' is reserved as a marker for transpiled ES2015 module.": {

```

Comment:
I don't think the `Identifier expected` is useful in any way. `__esModule` is an identifier. 😄